fss/src/app/children/tabs/interactions-form.component.html

```diff
@@ -247,5 +247,28 @@
                 </tr>
             </tbody>
         </table>
+
+        <input #interactionFileInput [hidden]="true" id=interaction-file-select type="file" name="interactions" ng2FileSelect [uploader]="fileuploader"/>        
+        <!-- <button mat-raised-button (click)="openFileSelect()">Upload...</button> -->
+        <mat-list dense>
+            <mat-list-item>Attachments
+                <button mat-icon-button>
+                    <mat-icon svgIcon="upload" (click)="openFileSelect()"></mat-icon>
+                </button>
+            </mat-list-item>
+            <mat-list-item *ngFor="let file of uploaded_files">{{file}}
+                <!-- <button mat-icon-button>
+                    <mat-icon svgIcon="download" (click)="downloadFile(file)"></mat-icon>
+                </button> -->
+                <a [href]="this.restService.getInteractionFileDownloadUrl(interaction.id, file)" download>
```

Comment:
Not really a review comment just for my own learnings. Why do we need to say this.restService as apposed to just restService? It seems that in other instances the "this" namespace is implied. Are services special cased?